Patch: 72_fblast_add_shebang.patch
Author: Thijs Kinkhorst <thijs@debian.org>
This script may be called directly from the command line, is set as executable,
thus needs a shebang line.
Index: tests/fblast.py
===================================================================
--- tests/fblast.py.orig	2006-10-17 10:08:34.000000000 +0200
+++ tests/fblast.py	2006-10-17 10:09:07.000000000 +0200
@@ -1,3 +1,4 @@
+#! /usr/bin/env python
 """Throw email at Mailman as fast as you can.
 
 This is not a unit test, it's a functional test, so you can't run it within
